opentelemetry-sdk/src/opentelemetry/sdk/util.py

```diff
@@ -46,7 +46,7 @@ class BoundedList(Sequence):
 
     def __init__(self, maxlen):
         self.dropped = 0
-        self._dq = deque(maxlen=maxlen)
+        self._dq = deque(maxlen=maxlen)  # type: deque
```

Comment:
Pretty surprising that you need these, but I see that you do.

Reply:
I guess the reason is that I should really be using something like `deque[T]` so that mypy knows the type of the contained items. Of course doing that properly would require making BoundedList a generic.

Reply:
That makes sense, and does seem like overkill at the moment to me.